src/sentry/static/sentry/app/actionCreators/indicator.jsx

```diff
@@ -69,12 +87,20 @@ export function saveOnBlurUndoMessage(change, model, fieldName) {
 
   if (!label) return;
 
+  let prettifyValue = val => prettyFormString(val, model, fieldName);
+  let showChangeText = model.getDescriptor(fieldName, 'formatMessageValue') !== false;
```

Comment:
I would add a comment here saying we only hide change text when `formatMessageValue` is explicitly `false`.